reqwest-retry/Cargo.toml

```diff
@@ -10,22 +10,22 @@ keywords = ["reqwest", "http", "middleware", "retry"]
 categories = ["web-programming::http-client"]
 
 [dependencies]
-reqwest-middleware = { version = "^0.1.6", path = "../reqwest-middleware" }
+reqwest-middleware = { version = "^0.1.6" }
```

Comment:
we need to remove the path when releasing because otherwise, it will point to `reqwest-middleware v 0.1.7-alpha.0`

we will set  to `reqwest-middleware = { version = "^0.1.7-alpha.0", path = "../reqwest-middleware" }` in a future PR